index.html

```diff
@@ -18,13 +17,17 @@
             <header class="navbar">
                 <a href="#">RODCHAK</a>
 
-                <nav><a href="#">
-                        <span class="material-symbols-outlined mobile">menu</span></a>
-                    <ul class="menu_bar desktop">
-                        <li> <a href="#">Portfolio</a></li>
-                        <li><a href="#">About</a></li>
-                        <li> <a href="#">Contact</a></li>
+                <nav>
+                    <ul class="menu_bar">
+                        <li class="nav-item"><a href="#Works_Section" class="nav-link">Portfolio</a></li>
+                        <li class="nav-item"><a href="#About_Myself" class="nav-link">About</a></li>
+                        <li class="nav-item"><a href="#Contact_form" class="nav-link">Contact</a></li>
                     </ul>
+                    <div class="hamburger">
+                        <span class="bar"></span>
+                        <span class="bar"></span>
+                        <span class="bar"></span>
+                    </div>
                 </nav>
             </header>
             <div class="background_content">
```

Comment:
- [ ] Good job so far, your project is almost completed. I noticed that you miss the `blur effect` on the **Background** of the `menu` to match the Figma design also make sure to add the correct spacing around the `Close_Menu(top && right)` button.

| ### Figma Design | Your Design |
| ---------------------- | ------------------ |
| ![Screenshot from 2022-07-19 21-05-28](https://user-images.githubusercontent.com/83042742/179830313-4b261056-4d6c-4dcd-8308-f937463ab224.png) | ![Screenshot from 2022-07-19 21-04-22](https://user-images.githubusercontent.com/83042742/179830411-624ef66b-55fa-4f19-acf7-b0ece321376a.png) |